test/systemtests/trigger_test.go

```diff
@@ -276,10 +276,10 @@ func (s *systemtestSuite) TestTriggerNetPartition(c *C) {
 
 			// flap the control interface
 			c.Assert(node.bringDownIf("eth1"), IsNil)
-			time.Sleep(50 * time.Second) // wait till sessions/locks timeout
+			time.Sleep(25 * time.Second) // wait till sessions/locks timeout
 			c.Assert(node.bringUpIf("eth1", nodeIP), IsNil)
 
-			time.Sleep(20 * time.Second)
+			time.Sleep(23 * time.Second)
```

Comment:
@erikh we do have the polling in place. All I did was adjust the base delay. I added 3 sec here and removed 25 sec above this.